Test/test0/SplitOnEveryAssert.bpl

```diff
@@ -0,0 +1,42 @@
+// Don't use %parallel-boogie so the trace output is more predictable
+// RUN: %boogie /trace "%s" > "%t"
+// RUN: %OutputCheck --file-to-check "%t" "%s"
+// CHECK:      checking split 1/12, .*
+// CHECK:      checking split 2/12, .*
+// CHECK:      checking split 3/12, .*
+// CHECK:      checking split 4/12, .*
+// CHECK:      --> split #4 done,  \[.* s\] Invalid
+// CHECK:      checking split 5/12, .*
+// CHECK:      checking split 6/12, .*
+// CHECK:      checking split 7/12, .*
+// CHECK:      checking split 8/12, .*
+// CHECK:      checking split 9/12, .*
+// CHECK:      checking split 10/12, .*
+// CHECK:      checking split 11/12, .*
+// CHECK:      checking split 12/12, .*
+// CHECK-L: SplitOnEveryAssert.bpl(30,5): Error: This assertion might not hold.
+
+procedure {:vcs_split_on_every_assert} Ex() returns (y: int)
```

Comment:
Maybe adding another procedure to this regression test that does not have this attribute would be useful. I guess it would also be nice to have a regression that uses `/vcsSplitOnEveryAssert`.

Reply:
Yeah I was on the fence about that too - added!